### DISCUSSION TITLE
Is there a good terminal emulator to run textual apps on Windows 7?

### DISCUSSION THREAD

**Discussion by labo-China** [Textualize/textual#4245] [General]
I ran into some compatibility problems with Windows 7.
I have tried serveral terminal emulators including WezTerm, alacritty, and ConEmu.
Among them some are unsupported on Windows 7, some are unable to process control sequences. 
ConEmu are able to display it but the effect is terrible and it cannot detect window size changes.

Is there a emulator to make it run on Windows 7? or Textual is simply not intended to ensure the compatibility with it?
I completely know that Textual doesn`t promise that it will function correctly in a normal console. But I still want to get an answer.

Any help is appreciated!

**Comment by davep** [comment]
[This comment](https://github.com/Textualize/textual/issues/3378#issuecomment-1732598008) would seem to be relevant here.
